src/functions/getTestResultsByTesterStaffId.js

```diff
@@ -15,17 +22,25 @@ const getTestResultsByTesterStaffId = (event) => {
 
   if (!event.queryStringParameters) {
     if (event.queryStringParameters.testerStaffId && event.queryStringParameters.testStationPNumber && event.queryStringParameters.toDateTime && event.queryStringParameters.fromDateTime) {
+      console.log('Bad request in getTestResultsByTesterStaffId - missing required parameters');
+      if (subseg) subseg.addError('Bad request in getTestResultsByTesterStaffId - missing required parameters');
```

Comment:
Can we have this if statement block inside try block and such that subseg gets closed in finally block?

Reply:
fixed